client/components/Editor/ConfirmableDeleteButton.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Confirm } from 'semantic-ui-react';
+
+class ConfirmableDeleteButton extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            open: false
+        };
+    }
+
+    render() {
+        const { onConfirm, what } = this.props;
```

Comment:
Heh i like the name `what`, but I find myself still asking "what is in this variable?". Perhaps we can make this more descriptive? Something like `itemType`?

Reply:
I'm not opposed to that change, can you add another commit?